coderedcms/templates/coderedcms/pages/article_page.html

```diff
@@ -5,8 +5,10 @@
 {% block content %}
     <article class="codered-article {%if self.cover_image %}has-img{% endif %}">
         {% if self.cover_image %}
+        {% block cover_image %}
```

Comment:
Blocks generally should go outside of if/else statements. Because if the `if` is not true, it will cause weird behavior.